buildpipeline/Core-Setup-CentOS-x64.json

```diff
@@ -3,6 +3,43 @@
     {
       "enabled": true,
       "continueOnError": false,
+      "alwaysRun": true,
```

Comment:
If cleanup is running regularly, you shouldn't be in a position where running cleanup is a prereq to building.  I also don't think we want a cleanup failure to block running the build, so I'd prefer that cleanup stuff happened at the end.  The agent cleanup stuff happens at the end in Windows builds, I don't see it enabled for Linux.